sentry-android-core/src/main/java/io/sentry/android/core/LifecycleWatcher.java

```diff
@@ -101,6 +108,29 @@ public void onBackground() {
     scheduleEndSession();
 
     addAppBreadcrumb("background");
+
+    if (enableLogFlushing) {
+      try {
+        scopes
+            .getOptions()
+            .getExecutorService()
+            .submit(
+                new Runnable() {
+                  @Override
+                  public void run() {
+                    scopes
+                        .getGlobalScope()
```

Comment:
`m` in theory there could be a different `SentryClient` instance on each scope type but that's probably more of an edge case. Romans suggestion above should get rid of that problem tho.